docs/reference/commandline/node_update.md

```diff
@@ -19,10 +19,31 @@ Update a node
 Options:
       --availability string   Availability of the node (active/pause/drain)
       --help                  Print usage
+      --label-add value       Add/update node labels (key1=value1,key2=value2)
```

Comment:
@dnephin for now, yes. We can decide to add the comma-separated format in a future release, but then probably for all flags at the same time to keep consistency (I can see the format being helpful)